src/ansys/fluent/core/solver/flobject.py

```diff
@@ -145,6 +146,15 @@ def obj_name(self) -> str:
             return self.fluent_name
         return self._name
 
+    @property
+    def python_name(self) -> str:
+        """Python name of this object.
+
+        By default, this returns the object's static name. If the object is a child of a
+        named object, the object's name is returned.
```

Comment:
But we can also return the class name prefixed with a dot. That doesn't look right.